terms.adoc

```diff
@@ -99,7 +97,7 @@ Proposal Phase ::
 The Proposal Phase is a Phase during which a Project Proposal is presented to the community and Membership at Large to solicit feedback.
 
 Release ::
-A Release is a specifically identifiable set of artifacts that are collectively assigned a durable tag in source control and distributed in a semantically versioned durable form of that collection intended by the developers of the project for third-party use.
+A Release is a collection of project artifacts intended for distribution beyond the project Developers.
```

Comment:
@waynebeaton I'm not happy with the mix of content and capitalization changes in one single commit. This makes it hard to follow.

FWIW, the capitalization changes look too broad and sometimes odd in the document.

Reply:
Understood. This is less than optimal. I should have done it with two passes.

The mixture does look odd. The original document had this mixture, and I removed it in 2015. It turns out that it is problematic to not have the mixed case when grouping this document with others written with the pseudo legal capitalization (e.g. IP Policy and Bylaws).

I took a pass through the definitions to make sure that they are consistent.

I'll send a case-insensitive red-line to the AC mailing list.